sql/top_manager_comeby_by_region.sql

```diff
@@ -0,0 +1,35 @@
+select
+    i.manager_id as '사원번호',
+    i.last_name as '이름',
+    i.annual_income as '연봉',
+    i.position_name as '직급명',
+    j.time as '입출입시간',
+    j.region as '지역',
+    j.record_symbol as '입출입구분'
+from
+    (
+        SELECT distinct(m.employee_id) as manager_id,
```

Comment:
```suggestion
        SELECT m.employee_id as manager_id,
```

distinct가 불필요하지 않나요?